src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -148,6 +148,7 @@ public String getQuestionTypeDisplayName() {
     public String getQuestionWithExistingResponseSubmissionFormHtml(
             boolean sessionIsOpen, int qnIdx, int responseIdx, String courseId,
             int totalNumRecipients,
+            boolean questionIsCompulsory,
```

Comment:
updated booleans. do we also need to update string literals '${questionIsCompulsory}' used inside templates ?